js/isomer.js

```diff
@@ -67,25 +72,215 @@ Isomer.prototype._translatePoint = function (point) {
 /**
  * Adds a shape or path to the scene
  *
- * This method also accepts arrays
+ * This method also accepts arrays.
+ * By default or if expertMode=0, shapes are drawn
  */
-Isomer.prototype.add = function (item, baseColor) {
+Isomer.prototype.add = function (item, baseColor, expertMode, name) {
```

Comment:
Do we keep the "name" parameter, which is to adress #34 ? (useful to remove/modify an element of the scene during an animation)